go.mod

```diff
@@ -0,0 +1 @@
+module github.com/saintfish/chardet
```

Comment:
It should match the import path `github.com/gogs/chardet`.

Reply:
Alright, I'll change the tests then.

Reply:
Thanks for making this PR!